src/soc/intel/alderlake/Kconfig

```diff
@@ -40,7 +40,7 @@ config SOC_INTEL_ALDERLAKE
 	select CPU_INTEL_COMMON_VOLTAGE
 	select SOC_INTEL_COMMON_ACPI_WAKE_SOURCE
 	select SOC_INTEL_COMMON_BASECODE
-	select SOC_INTEL_COMMON_BASECODE_RAMTOP
+	# select SOC_INTEL_COMMON_BASECODE_RAMTOP
```

Comment:
It should be disabled on SoC level for all ADL boards, not on the mainboard level?
How about upstreaming such a change?

Reply:
I doubt it will get merged. This "feature" is mostly for boot time optimization for Chromebooks.

If it causes problem on one board, then it will cause also problems on other boards as well most likely. We just haven't exercised memory compatibility that hard on MSI or ADL laptops.

Reply:
So enable for CHROME_OS only perhaps?
There must be a way for keeping chromebooks happy and not breaking other boards at the same time, that is acceptable for upstream.

Reply:
Ok, guarded with CHROMEOS